packages/openapi-sampler/src/samplers/StringSampler.ts

```diff
@@ -4,18 +4,31 @@ import RandExp from 'randexp';
 export class StringSampler implements Sampler {
   private readonly stringFormats = {
     'email': () => 'jon.snow@targaryen.com',
+    'idn-email': () => 'джон.сноу@таргариен.укр',
     'password': (min: number, max: number) =>
       this.adjustLength('p@$$w0rd', min, max),
     'date-time': () => '2021-12-31T23:34:00Z',
     'date': () => '2021-12-31',
+    'time': () => '23:34:00Z',
```

Comment:
```suggestion
    'time': () => '23:34:00',
```

imo it's not correct that `time` contain timezone info and `date` is not (though it's done in this way in "parent" repo).

https://github.com/OAI/OpenAPI-Specification/issues/845#issuecomment-378139730 defines `time` as `partial-time` from RFC3339](http://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14):
 `partial-time = time-hour ":" time-minute ":" time-second [time-secfrac]`
i. e.  optional milliseconds and no 'Z'

Reply:
OAS is compliant with JSON Schema Draft 2020-12 (https://swagger.io/specification/#data-type-format), which defines the `time` format as a `full-time` as specified in https://json-schema.org/draft-07/json-schema-validation.html#rfc.section.7.3.1. You can find more information about this duration in https://www.rfc-editor.org/rfc/rfc3339#section-5.6.

Reply:
Thanks for clarification!  Indeed it should be `full-time` instead of `partial-time` despite of comment referenced above. Proper link to corresponding section in "JSON Schema Draft 2020-12" https://json-schema.org/draft/2020-12/json-schema-validation.html#section-7.3.1-4.6